collectionchallenge.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+$db = new PDO('mysql:host=db; dbname=pokemon', 'root', 'password' );
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare("SELECT `stats`.`name`, `stats`.`type1`, `stats`.`type2`, `stats`.`hp`, `stats`.`attack`, `stats`.`defense`, `stats`.`spAttack`, `stats`.`spDefense`, `stats`.`speed` FROM `stats`;");
```

Comment:
Could use * if you're selecting all the fields from your table but otherwise looks good :)

Reply:
Decided to go without * as best practice is not to use it in end code and it would make extra work for me to go back in and change it